config/src/config/mod.rs

```diff
@@ -400,7 +414,6 @@ mod test {
         let root_dir = RootPath::new(expected.test.as_ref().unwrap().temp_dir().unwrap());
         let path = root_dir.full_path(&PathBuf::from("node.config.toml"));
         expected.save(&path).expect("Unable to save config");
-
```

Comment:
why do you hate whitespace?

Reply:
I actually really like whitespace.

I spent a lot of time putting printlns places to see what was going on, I may have deleted this line.